### PR TITLE
Array value delimiter

### PR DESCRIPTION
**Pull request by dominikb**
Feature as described in #229 with using a configurable delimiter.

### CODE REVIEW

**Comment by iredmedia** [comment]
Is this going to be merged?

**Comment by francoism90** [comment]
Please merge this. :)

File: docs/features/filtering.md [hunk 1]
```diff
@@ -228,3 +228,35 @@ QueryBuilder::for(User::class)
     ])
     ->get();
 ```
+
+## Array value delimiter
```
Comment (freekmurze) [inline docs/features/filtering.md]: Nice work updating the readme 👍

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**